src/get-bounding-client-rect.js

```diff
@@ -48,6 +47,10 @@ function getBoundingClientRect() {
  * @return {boolean}
  */
 function shouldInstall(win) {
+  if (!win.document) {
```

Comment:
Minor: Add `// Don't install in no-DOM environments e.g. worker.`.

Reply:
This is resolved in e090680 . Thanks for the suggestion.